vertical-pod-autoscaler/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG BASE_IMAGE
+
+FROM --platform=$BUILDPLATFORM golang:1.19 AS builder
+WORKDIR /go/src/k8s.io/autoscaler
+COPY . .
+ARG TARGETARCH
+ENV GO111MODULE "on"
+ENV GOARCH=${TARGETARCH}
+
+WORKDIR /go/src/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender
+RUN env CGO_ENABLED=0 GOARCH=${TARGETARCH} GOOS=linux go build -o vpa-recommender -buildvcs=false --ldflags "-s"
```

Comment:
with the introduction of the before script there https://github.com/DataDog/compute-delivery/blame/master/gitlab-pipeline-templates/v2/compute-delivery.yml#L137
it should work without `-buildvcs=false`